addons/web/static/src/js/views/list/list_renderer.js

```diff
@@ -205,12 +207,17 @@ var ListRenderer = BasicRenderer.extend({
      *
      * @private
      * @param {any} aggregateValues
+     * @param {Boolean} isHeader indicates wheter the groups rendered are on the header or on the footer
      * @returns {jQueryElement[]} a list of <td> with the aggregate values
      */
-    _renderAggregateCells: function (aggregateValues) {
+    _renderAggregateCells: function (aggregateValues, isHeader) {
         var self = this;
-        return _.map(this.columns, function (column) {
-            var $cell = $('<td>');
+
+        return _.map(this.columns, function (column, index) {
+            if (isHeader && index < headingColumnsToSkipInGroups-1 && self.columns.length > headingColumnsToSkipInGroups) {
```

Comment:
too long, split this line.
self.columns.length > headingColumnsToSkipInGroups necessary? Why?